src/prototype/Scenario2.tscn

```diff
@@ -0,0 +1,164 @@
+[gd_scene load_steps=7 format=2]
```

Comment:
I don't understand why you did this. I thought we're going to reuse the same arena from `Main.tscn` for all of the other things we're prototyping.

Reply:
I think you should discard it. More code means more maintenance. 

If you're planning to add ramps/traps/etc. by hand, which is a good idea, you should instead make this a subclass of the empty arena `Main.tscn`